app/src/main/java/com/jerboa/ui/components/common/AppBars.kt

```diff
@@ -112,99 +111,87 @@ fun SimpleTopAppBarPreview() {
     }
 }
 
-@Composable
-fun BottomAppBarAll(
+fun NavigationSuiteScope.adaptiveNavigationBar(
     selectedTab: NavTab,
-    onSelect: (NavTab) -> Unit,
+    onSelectTab: (NavTab) -> Unit,
     userViewType: UserViewType,
     unreadCounts: Long,
     unreadAppCount: Long?,
     unreadReportCount: Long?,
     showTextDescriptionsInNavbar: Boolean,
 ) {
-    // Check for preview mode
-    if (LocalContext.current is Activity) {
-        val window = (LocalContext.current as Activity).window
-        val colorScheme = MaterialTheme.colorScheme
-
-        DisposableEffect(Unit) {
-            window.navigationBarColor = colorScheme.surfaceContainer.toArgb()
-
-            onDispose {
-                window.navigationBarColor = colorScheme.background.toArgb()
-            }
+    for (tab in NavTab.getEntries(userViewType)) {
+        val selected = tab == selectedTab
+        val iconBadgeCount = when (tab) {
+            NavTab.Inbox -> unreadCounts
+            NavTab.RegistrationApplications -> unreadAppCount
+            NavTab.Reports -> unreadReportCount
+            else -> null
         }
-    }
-    // If descriptions are hidden, make the bar shorter
-    val modifier = if (showTextDescriptionsInNavbar) Modifier else Modifier.navigationBarsPadding().height(56.dp)
```

Comment:
We can't mess with the navbar sizes anymore, since its adaptive. We can still hide the text at least (that's working).

Reply:
Was never huge fan as its a bit fragile. But I can see why users wanted it.